### ISSUE TITLE
EF attribute [NotMapped] not processed

### ISSUE THREAD

**Issue by maurei** [christiandelbianco/monitor-table-change-with-sqltabledependency#117]
Hi,

The following error

```
"Model with columns having same name."
```
is produced in the following scenario

```c#
public class Client 
{
    ...
    ...
    [JsonProperty("Number"), NotMapped]
    public int Number
    {
        get { return ParseOrReportBadNumber(StringNumberInDatabase); }
        set { StringNumberInDatabase = value.ToString(); }
    }
    [Column("Number")]
    public string StringNumberInDatabase { get; set; }
    ..
    ..
}
```
Expected behaviour: not having this error, as `[NotMapped]` implies this property is not in the database.

Usecase: I have a legacy database in which the `Number` column is a `string`, but in all client apps are  treating it as `int`. There are some values in the db that cannot trivially be converted to int, hence `ParseOrReportBadNumber()` that initiates some additional action.

Workaround:  fairly easy: changing `public int Number` such that it does not overlap with the argument of the  `[Column(..)]` attribute.

**Comment by christiandelbianco** [comment]
Yes, managing the **[NotMapped]** attribute is a good idea.

**Comment by christiandelbianco** [comment]
**[NotMapped]** attribute available with version **8.5.3**.
